src/platform/api (legacy)/index.js

```diff
@@ -13,75 +13,72 @@ app.get('/', (req, res) => {
   (async () => {
     const webpageUrl = 'https://www.eventbrite.ca';
 
-    const browser = await puppeteer.launch({ headless: true });
+    const browser = await puppeteer.launch({ headless: false });
     const page = await browser.newPage();
     await page.setViewport({
       width: 1920,
       height: 1080,
     });
 
     await page.goto(webpageUrl, { waitUntil: 'networkidle2' });
+
+    await timeout(3000);
+
     await page.type('#locationPicker', 'Toronto');
+
     await page.keyboard.press('Enter');
-    await timeout(2000);
+
+    await timeout(3000);
 
     eventsArray = await page.evaluate(() => {
       const events = [];
       for (let i = 1; i < 9; i += 1) {
         console.log(i);
         const eventTitle = document.querySelector(
```

Comment:
hmm, let's remove these changes made to the legacy server

Reply:
Resolved...everything has been reverted upto my knowledge, let me know if you find something else 😬